### ISSUE TITLE
[Bug]: Could not connect: Timeout connecting to server

### ISSUE THREAD

**Issue by tapliar** [RedisInsight/RedisInsight#751]
**Preconditions** (Any important steps we need to know)
EKS v1.20.15-eks-a64ea69 cluster
RedisInsight pod running latest image
AWS PrivateLink to allow EKS, hosted in AWS account B, to connect to redis cluster in AWS account A

**Steps to reproduce** (How to reproduce what you found step by step)
1. Open RedisInsight which is hosted in EKS
2. Add Redis database (using VPC endpoint as host)
![add-cluster](https://user-images.githubusercontent.com/15873664/190919603-d9022602-c4c0-43ce-86a2-0cd7bb0fcd68.png)
3. Select seed nodes to connect with
![seed-nodes](https://user-images.githubusercontent.com/15873664/190919615-91426981-0317-45c9-8bc1-928576f4de47.png)

**Actual behavior** (A short description of what you found)
Unable to connect to the seed nodes which results in a timeout error
![seed-error](https://user-images.githubusercontent.com/15873664/190919629-51a719a4-ea86-4e10-bc29-830f5704598b.png)

**Expected behavior** (A short description of what you expected to find)
Confirmed that this is not an issue with AWS PrivateLink by deploying a pod which had the redis-cli install and successfully connected to the Redis database
```
/ # redis-cli -h vpce-xxx-xxx.vpce-svc-xxx.eu-west-1.vpce.amazonaws.com -c --tls -p 6379
vpce-xxx-xxx.vpce-svc-xxx.eu-west-1.vpce.amazonaws.com:6379>
```
As the cluster endpoint is a different interface to the seed nodes, I suspect that this is a VPC issue, as the RedisInsight pod and the Redis database are not within the same VPC.
Is there a configuration option that could be used to bypass the "select seed nodes" step?

**Comment by ViktarStarastsenka** [comment]
@tapliar , thanks for reporting this, we will investigate it.

**Comment by GnaneshKunal** [comment]
Hey @tapliar 


Currently "Select seed nodes" step is mandatory for cluster databases. The timeout error seems to be with connectivity issues with one of the nodes being down or not rechabable.


In the machine where redisinsight is running, run `redis-cli` with cluster mode and run some cluster operations and see whether the other node is reachable. If one of the node is erroring out, try to connect to the individual node if possible or try modifying the other node's configuration for connectivity. 

Let us know if connectivity is the issue.

**Comment by tapliar** [comment]
Hey @GnaneshKunal 

I did deploy a pod with the `redis-cli` installed on the same EKS cluster as the RedisInsight pod. I ran `redis-cli` with cluster mode and TLS, and I was able to connect to the Redis database, and I could run some commands.

As I suspected that this was a VPC issue, I decided to deploy RedisInsight on ECS Fargate in the same AWS account and within the same VPC as the Redis database instead. This time, the "select seed nodes" step succeeded with no error, and I was able to add the Redis database.
